CHANGELOG.md

```diff
@@ -30,10 +30,20 @@
   union other_employees
   ```
 
-- Any other `feat:` commits?
+The following need updated pages in the documentation:
+
+- feat: Playground queries sample database (#1305)
+- feat: comment that displays the compiler version used to generate the SQL
+  (#1322)
+- feat: `in` operator (#1330)
```

Comment:
If anyone knows a good way of generating changelogs with both a PR reference and the contributor, that would be v useful — anything we can do to raise the status of contributors is great (even though this time it's mostly @aljazerzen...).

(but without adding a burden on contributors themselves, e.g. `changie`)